en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+
+        | **Field**                  | **Value**            |
+        |----------------------------|----------------------|
+        | **Country Code**           | `country`            |
+        | **Response Variable Name** | `populationByCountry`|
+
+    5. Click **Save**.
+
+    6. Now let’s extract the population value from the response, calculate the population in millions, and store it in a variable. Follow this procedure:
+
+        1. Click the last **+** icon in the low-code diagram.
+
+        2. Under **Statements**, select **Variable** and enter details as follows:
+
+            | **Field**      | **Value**                                         |
+            |----------------|---------------------------------------------------|
+            | **Type**       | `int`                                             |
+            | **Name**       | `populationMillions`                              |
+            | **Expression** | `(populationByCountry[0]?.value ?: 0) / 1000000`  |
+
+        3. Click **Save**.
+
+6. Now let’s calculate the total COVID-19 case count per million in the population based on the COVID-19 statistics and the population data you have retrieved. Follow this procedure:
+
+    1. Click the last **+** icon in the low-code diagram.
+
+    2. Under **Statements**, select **Variable** and enter details as follows:
+
+        | **Field**      | **Value**                                    |
+        |----------------|----------------------------------------------|
+        | **Type**       | `decimal`                                    |
+        | **Name**       | `totalCasesPerMillion`                       |
+        | **Expression** | `<decimal>(totalCases / populationMillions)` |
+
+    3. Click **Save**.
+
+7. To build the JSON payload to be sent as the response and then send the response, follow this procedure:
+
+    1. To build the `json` payload with data of the total cases per million in the population, add a variable.
+
+        Click the last **+** icon in the low-code diagram and click **Variable**. Then enter information as follows:
+
+         | **Field**      | **Value**                     |
+         |----------------|-------------------------------|
+         | **Type**       | `json`                        |
+         | **Name**       | `payload`                     |
+         | **Expression** | `{country : country, totalCasesPerMillion : totalCasesPerMillion}` |
+
+    2. Click **Save**.
+
+    3. To respond with the JSON payload, add a `Return` statement.
+
+        Click the last **+** icon in the low-code diagram and click **Return**.
+
+        In the **Return Expression** field, enter `payload`.
+
+        Save the information.
+
+Now you have completed designing the `CovidStatus` service.
+
+The low-code diagram looks as follows:
+
+ ![Add service construct](../assets/img/tutorials/rest-api/low-code-diagram.png){.cInlineImage-full}
+
+The code view looks as follows:
```

Comment:
do we need this? Shall we add an image with both views side by side? and maybe have the code in a file in our repo, so that they can take a look if needed. We can mention, that and add a link.

Reply:
The practice we decided on was to have the code in text format so that a user who is not interested in trying out all the steps can copy and paste it into the editor.